<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParamNISearcherWdgt</class>
 <widget class="QWidget" name="ParamNISearcherWdgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Working directory</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="tbChangeProjectDir">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../../Matilda-units/gui/image-and-font/res.qrc">
           <normaloff>:/katynko/svg/document-open.svg</normaloff>:/katynko/svg/document-open.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="lblProjectDir">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblNIs">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QToolButton" name="tbNiHistory">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../../Matilda-units/gui/image-and-font/res.qrc">
           <normaloff>:/katynko/svg/edit-find.svg</normaloff>:/katynko/svg/edit-find.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="leNIs">
         <property name="placeholderText">
          <string>NIs space separated</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <layout class="QVBoxLayout" name="vl4wdgt"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../Matilda-units/gui/image-and-font/res.qrc"/>
 </resources>
 <connections/>
</ui>
